pkg/distributor/distributor_test.go

```diff
@@ -620,7 +706,7 @@ func TestPush_QuorumError(t *testing.T) {
 	ingesters[2].failResp.Store(httpgrpc.Errorf(429, "Throttling"))
 
 	for i := 0; i < numberOfWrites; i++ {
-		request := makeWriteRequest(0, 30, 20)
+		request := makeWriteRequest(0, 30, 20, 0)
```

Comment:
Should we test for the native histogram case here as well?

Reply:
I don't know if it is nencessary since histogram doesn't change anything related to quorum. And we mock response here anyway.

Reply:
Updated the test to include histogram writes.